glib-macros/src/lib.rs

```diff
@@ -374,3 +375,32 @@ pub fn downgrade(input: TokenStream) -> TokenStream {
     let input = parse_macro_input!(input as DeriveInput);
     downgrade_derive::impl_downgrade(input)
 }
+
+/// An attribute macro for wrapping the function's body  in a `once_cell::sync::Lazy`.
+///
+/// The macro is mostly useful for `ObjectImpl` properties/signals definitions
+///
+/// Example:
+///
+/// Without using the macro
+/// ```rust
+/// pub fn static_func() -> &'static [String] {
+///     static items: Lazy<Vec<String>> = Lazy::new(|| {
+///         vec!["item".to_string(), "item2".to_string()]
+///     });
+///     items.as_ref()
+/// }
+/// ```
+/// With the macro
+/// ```rust
+/// #[static_]
```

Comment:
Maybe `lazy_static` would be a more descriptive name

Reply:
(note that this is basically what the `lazy_static!` macro did, just in attribute position now...)

Reply:
The name was just there to get something working first. Renamed it to `lazy_static`